tests/unit/test.spec.js

```diff
@@ -0,0 +1,36 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import AddMemorialForm from '@/components/AddMemorialForm.vue'
+import VueI18n from 'vue-i18n'
+
+describe("addmemorial testl", () => {
+    let localVue = createLocalVue()
+    localVue.use(VueI18n)
+    test = {
+      name: 'kevin smith',
+      gender: 'male',
+      photo_upload: 'no',
+      photo_base64: 'nan',
+      birth_date: '25-11-1954',
+      passing_date: '25-11-2020',
+      location: 'jaipur::rajasthan',
+      prompt: 'none',
+      prompt_response: 'none'
+  }
+    const i18n = new VueI18n({
+      locale: 'en',
+      fallbackLocale: 'en'
+      
+    })
+    
+    const wrapper = shallowMount( AddMemorialForm, {i18n,localVue});
+
+  it("renders successfully", () => {
+    mocks: {
+      $t: (msg) => msg
+    }
+      
+      wrapper.vm.test();
+      expect(wrapper.vm.memorial).toStrictEqual(test); 
```

Comment:
The way this test is written right now, you're testing that the object inside of memorial is the same as this test file. You're not testing that the *output* of the rendering looks the way you want it to.

You want to test the rendering piece by piece as in this example here:
https://vuejs.org/v2/cookbook/unit-testing-vue-components.html

Reply:
by piece by piece, you mean to test value for each property seperately? like for e.g. name: '',  and name: 'kevin smith', should be tested using seperate line. alright will do.